metals/src/main/scala/scala/meta/metals/MetalsServices.scala

```diff
@@ -411,7 +413,25 @@ class MetalsServices(
       response
     case SbtConnect =>
       Task {
-        connectToSbtServer()
+        val path = cwd.resolve("project").resolve("build.properties")
+        val input = Files.newInputStream(path.toNIO)
+        val sbtVersion: Option[String] =
+          Try {
+            val props = new Properties()
+            props.load(input)
+            Option(props.getProperty("sbt.version"))
+          }.toOption.flatten
+        input.close()
+
+        sbtVersion match {
+          case Some(ver) if ver.startsWith("0.13") || ver.startsWith("1.0") =>
```

Comment:
are we confident enough there are no `0.12` around? I've worked on such a build recently :D 

How about 

```scala
ver.startsWith("0.")
```

Reply:
Good point. On sbt-metals won't work on 0.12 anyway, but it doesn't hurt to cover more cases with this warning.